src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java

```diff
@@ -36,16 +36,21 @@ public int getColumnCount(String tableName) {
     try {
       Statement informationSchemaStatement = ConnectionProvider.getInstance().getConnection()
           .createStatement();
-
-      ResultSet resultCount = informationSchemaStatement.executeQuery(sqlSchemaQuery);
-      count = resultCount.getMetaData().getColumnCount();
-      log.info("Count is " + count);
+      try {
```

Comment:
why these many nested trys

Reply:
Able to close first result set then statement and connection.